src/cclib/parser/utils.py

```diff
@@ -143,6 +165,29 @@ def __init__(self):
             self.number[self.element[i]] = i
 
 
+class Splitter:
+    """Split a line based not on a character, but a given number of field
+    widths.
+    """
+
+    def __init__(self, widths):
+        self.start_indices = [0] + list(accumulate(widths))[:-1]
+        self.end_indices = list(accumulate(widths))
+
+    def split(self, line):
+        """Split the given line using the field widths passed in on class
+        initialization.
+        """
+        elements = [line[start:end].strip()
+                    for (start, end) in zip(self.start_indices, self.end_indices)]
+        for i in range(1, len(elements)):
+            if elements[-1] == '':
```

Comment:
Ok. The reason is to handle lines with empty regions, which come out like:

```python
['60', 'H', '10', 's', '0.14639', '0.00000', '0.00000', '-0.00000', '-0.00000', '0.00000']
['1', 'C', '1', 's', '-0.00000', '-0.00000', '0.00000', '', '', '']
```

and are then removed.